var mensaje = 'Prueba try-catch';

try {
    alert(mensaj);
} catch (error) {
   alert('ha ocurrido un error: ' + error) ;
}